src/host/main.cpp

```diff
@@ -165,6 +156,34 @@ int main(int argc, char** argv)
     "set to a significantly lower value than --raft-election-timeout-ms.",
     true);
 
+  size_t raft_election_timeout = 5000;
+  app.add_option(
+    "--raft-election-timeout-ms",
+    raft_election_timeout,
+    "Raft election timeout in milliseconds. If a follower does not receive any "
+    "heartbeat from the leader after this timeout, the follower triggers a new "
+    "election.",
+    true);
+
+  size_t pbft_view_change_timeout = 5000;
+  app.add_option(
+    "--pbft_view-change-timeout-ms",
+    pbft_view_change_timeout,
+    "Pbft view change timeout in milliseconds. A backup that receives a "
```

Comment:
Minor: Could we phrase it in a similar way as the Raft election timeout description for consistency? e.g. "If a backup does not receive the pre-prepare message for a request forwarded to the primary after this timeout, the backup triggers a new view change".

Reply:
Sure :)